app/errors.py

```diff
@@ -25,5 +26,6 @@ def generic_error_handler(e):
 
 
 for code in range(400, 599):
-    main.app_errorhandler(code)(generic_error_handler)
-    callbacks.app_errorhandler(code)(generic_error_handler)
+    if code in default_exceptions:  # flask complains if we attempt to register a handler for status code its unaware of
+        main.app_errorhandler(code)(generic_error_handler)
+        callbacks.app_errorhandler(code)(generic_error_handler)
```

Comment:
Can we just do `for code in default_exceptions:` and ditch the `if`?

Reply:
Mmmmmmmmmaybe... as long as we're 100% certain that werkzeug will only ever put 400 -> 599 status codes in there?

Reply:
If they do put extra ones in, we should probably handle them 😹